### DISCUSSION TITLE
Install help: blank page on fresh install

### DISCUSSION THREAD

**Discussion by adamalfath** [Part-DB/Part-DB-server#655] [Q&A] [answered]
Disclaimer: I'm complete noob regarding web stuff, but I could follow the technical steps.

So I'm on debian 12 machine, following every step of the installation guide, have MariaDB and Apache config ready, requirements check have all green except one yellow regarding Opcache improvement. But when opening the url all I get is a blank screen and with source below
```
<?php

use App\Kernel;

require_once dirname(__DIR__).'/vendor/autoload_runtime.php';

return function (array $context) {
    return new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']);
};
```

Can anybody give me a clue what I might be missed?

**Comment by jbtronics** [comment, marked as answer]
This is the content of the `index.php` file. If you are seeing this in your browser (instead of Part-DB), then your webserver does not correctly parse PHP files (and pass them to PHP).

You have to check your apache config, that PHP is configured.

**Reply by adamalfath** [reply]
Thanks for the insight.

Things I do differently from the guide is by installing php 8.3, I need to enable php-fpm config on apache. Now the webpage is loaded correctly.
